Made editor buttons not transparent (on hover)

Libter opened the pull request.

jjeffryes commented:
Don't change this in the medium editor css, we should leave that as-is since it's an external library.

Instead put an override in the obBase.css file. There are some override classes for libraries towards the bottom of the file.

What problem is this solving? The buttons don't seem to be transparent in master.

Libter committed:
Removed transparency on hover from editor buttons (another solution)

Libter commented:
@jjeffryes in master buttons on hover are transparent and text under them is visible.

Before fix:
![image](https://cloud.githubusercontent.com/assets/5753763/14586406/4bd88816-0497-11e6-9f9d-958f7b18816c.png)

After:
![image](https://cloud.githubusercontent.com/assets/5753763/14586438/7f7860aa-0498-11e6-8715-730065f2d953.png)

I removed previous commit and added fix to obBase.css.
